index.bs

```diff
@@ -839,6 +838,21 @@ When this method is invoked, the user agent MUST execute the following algorithm
                 1.  Let |attestationObject| be a new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the
                     bytes of <code>|credentialCreationData|.[=attestationObjectResult=]</code>'s value.
 
+                1.  Let |attestationPreference| be the value of |options|.{{MakePublicKeyCredentialOptions/attestation}}.
+                    <dl class="switch">
+                    <dt>If the value of |attestationPreference| is "none" (or it has been left unspecified),</dt>
+                    <dd>replace potentially identifying information (such as AAGUID and attestation certificates) 
+                        in the attested credential data and attestation statement with non-identifying versions of the same data.</dd>
```

Comment:
should use [=Attested credential data=] and [=attestation statement=].

s/identifying/uniquely identifying/
s/non-identifying/blinded/ ?